test/plugin/constants.rb

```diff
@@ -0,0 +1,422 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Constants used by unit tests for Google Cloud Logging plugin
+module Constants
+  include Fluent::GoogleCloudOutput::Constants
+
+  # generic attributes
+  HOSTNAME = Socket.gethostname
+
+  WRITE_LOG_ENTRIES_URI = 'https://logging.googleapis.com/v2beta1/entries:write'
```

Comment:
LOL I already moved it in my other branch of metadata agent work. Forgot to include it here though. I was trying to split this piece and keep this PR minimal. Just moved the location of `WRITE_LOG_ENTRIES_URI` in this PR as well.